.github/workflows/build.yml

```diff
@@ -43,3 +43,8 @@ jobs:
           tags: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.VERSION }}, ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:latest
           cache-from: type=registry,ref=${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:latest
           cache-to: type=registry,ref=${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:latest,mode=max
+      # Notify the top-level deploy project that an image was pushed.
+      - run: "curl -X POST -H \"Accept: application/vnd.github+json\" \
+             -H \"Authorization: token ${{ secrets.GITHUB_TOKEN }}\" \
+             https://api.github.com/repos/PhilanthropyDataCommons/deploy/actions/workflows/update-service-image.yml/dispatches \
```

Comment:
I hear your concern in the PR that this is somewhat dog wagging but I also appreciate that it may simply be the best we can do when using github actions for CD.

That said, rather than running a curl directly what do you think of using an existing github marketplace action.

This one seems to be the most starred / popular: https://github.com/marketplace/actions/repository-dispatch

Reply:
I haven't gotten this curl part to work just yet, and it seems ugly, so I will try the repository-dispatch approach, thanks!

Reply:
I also was hoping to avoid creating a PAT but that appears inevitable either way.